eXo/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -71,6 +71,11 @@
       "idiom" : "ipad",
       "filename" : "BD-2776_AppIcon_152x152.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ipad",
+      "size" : "83.5x83.5",
+      "scale" : "2x"
```

Comment:
You need to specify a file to fix the warning `Assets.xcassets: The app icon set "AppIcon" has an unassigned child.`